src/Reducers/reducer.js

```diff
@@ -0,0 +1,40 @@
+export const initialState = {
+    user: null,
+    playlists: [],
+    spotify: null,
+    discover_weekly: null,
+    top_artists: null,
+    playing: false,
+    item: null,
+    // token: 'BQA9NU-EwosKTyMDHjVH3A9D30MColXJkKZaJUpqWmhPG_nvnG83fme-xG5lJWp0rOwyqcV2Xgp1_p0_ORslCcjJnpzA7p7Ngxkx9V-VSc5LjkVoxjOdp-bq3LRyzQeuWiQSc-lL3Eg3HailcFFrYOjHTfcahDwodGSV8syYD5_Dz1h_IV4Z',
+};
+
+const reducer = (state, action) => {
+
+    switch (action.type){
+        case "SET_USER":
+            return {
+                ...state,
+                user: action.user,
+            };
+        case "SET_TOKEN":
+            return {
+                ...state,
+                token: action.token,
+            };
+        case "SET_PLAYLISTS":
+            return {
+                ...state,
+                playlists: action.playlists,
+            };
+        case "SET_DAILY_MIX":
+            return{
+                ...state,
```

Comment:
It is good practice to separate the scopes of the reducers, for example you can create another file called playlistReducer.js and in that have everything related to the playlist, another could be for searches, the intention is not to mix everything in the same file